### DISCUSSION TITLE
Connect Xbox 360 Kinect camera with SlicerIGT

### DISCUSSION THREAD

**Discussion by imilanov** [PlusToolkit/PlusLib#1060] [Q&A]
Hello,

I'm trying to connect Xbox 360 Kinect camera using existing PlusDeviceSet_Server_AzureKinect.xml configuration but without success. I also tried to modifie the config but still no success. With simple Logitech camera and Video for Windows video capture device.xml the video in Slicer working so it's not problem with connection or Plus Server. Do you know how I can create proper configuration for Kinect so I can include all sensors (like for Revopoint 3D Scanner) or this is not possible?

Thank you.

Regards,
Ivan

**Comment by Sunderlandkyl** [comment]
Are you building Plus from scratch? I don't think that the AzureKinect device is included in any of the downloadable packages.

**Reply by imilanov** [reply]
I think no. I downloaded and installed PlusApp-2.8.0.20191105-Win64.exe and now want to connect with Slicer3D and SlicerIGT module. I using instruction from https://www.slicerigt.org/wp/user-tutorial/ and now I'm on connecting hardware step (https://1drv.ms/p/s!AhiABcbe1DBygZ56ZCRIJk7CNVhfDw). On this page (http://perk-software.cs.queensu.ca/plus/doc/nightly/user/Configuration.html) I see configuration for AzureKinect(http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceAzureKinect.html). Goal is to achive this https://www.youtube.com/watch?v=lpnvMrpqjxs

**Comment by coldfire85** [comment]
Did you find a way yet?

**Comment by Biokareem** [comment]
did you find a solution ?

**Comment by adamrankin** [comment]
I don’t have a Kinect to test with. Is it that the device is unavailable/doesn’t work, or that the configuration is not correct?

edit: could you post a log of one of your attempts?
